### DISCUSSION TITLE
customize cover page with pdf2 plugin

### DISCUSSION THREAD

**Discussion by bing-10** [dita-ot/dita-ot#3988] [Q&A]
In the customization of pdf2 plugin, I found that the cover page margins are consistent with the chapter page, and I tried to modify the cover page in the defined attributes: `page-sequence.cover` and `__frontmatter` to remove the margins:

![image](https://user-images.githubusercontent.com/94583069/184473937-b7538ca5-cdcc-409d-8747-598e538812fc.png)

![image](https://user-images.githubusercontent.com/94583069/184474108-23f6bbbb-708e-4014-aad6-6961d6c0e8f0.png)

However, it did not work. The margins still keep the same with the chapter page.

Then I searched all the xsl files in the plugin to find the location where the page-sequence has been defined, but there is no result. 

Additionally, the information about **author**, **publisher** and **copyright** in the map cannot been transformed and printed in the final output. That is to say, such information did not appear in the cover page of the published PDF.

So how can I solve these problems?

**Comment by stefan-jung** [comment]
Hi, I'm not sitting in front of a PC, so I haven't looked this up for you. But the flow is the wrong page to do what you want to do. This is an XSL:FO to represent the text flow and not the appearance of a specific element. The solution is in the frontmatter files. Further on, regarding the metadata elements, check first if they have made it to the _MERGED.xml and the .fo file in the temp directory. So, set clean.temp="no" and then check those files. Afterwards, please come back to this thread and we take the next step

**Reply by bing-10** [reply]
Ok, thank you.

I have made it and found no metadata elements been generated, in both **_MERGED.xml** and **topic.fo** files.

but I found such definition of "front-matter" in **topic.fo**: 
```
<fo:page-sequence-master master-name="front-matter">
    <fo:repeatable-page-master-alternatives>
        <fo:conditional-page-master-reference master-reference="front-matter-first" odd-or-even="odd" page-position="first"/>
        <fo:conditional-page-master-reference master-reference="front-matter-last" odd-or-even="even" page-position="last"/>
        <fo:conditional-page-master-reference master-reference="front-matter-odd"/>
    </fo:repeatable-page-master-alternatives>
</fo:page-sequence-master>
```
What should I do for the next step?

**Reply by stefan-jung** [reply]
Hi, sorry for the late reply. Probably the problem is the block-container. You can make it visible by setting the background-color. Have you tried that?

**Reply by bing-10** [reply]
Yes, I have tried it at first and found that the map title has the same margins (start and end) as the chapter content.
